server/controllers/articlesController.js

```diff
@@ -0,0 +1,206 @@
+import db from '../models';
+import generateUniqueSlug from '../helpers/generateUniqueSlug';
+import checkEmptyString from '../helpers/checkEmptyString';
+
+const { Article, Category } = db;
+/**
+* @description class will implement CRUD functionalities for articles
+*
+* @class ArticleController
+*/
+class ArticlesController {
+  /**
+   * @param {object} req - Request sent to the router
+   * @param {object} res - Response sent from the controller
+   * @param {object} next - Error handler
+   * @param {object} article
+   * @returns {object} - object representing response message
+   */
+  static async createArticle(req, res, next) {
+
+    if (checkEmptyString(req)) {
+      return res.status(400).send({
+        success: false,
+        message: 'Some fields are missing',
+      });
+    }
+    const { categoryId } = req.body;
+
+    const articleSlug = generateUniqueSlug(req.body.title);
+    try {
+      const getCategory = await Category.findOne({
+        where: { id: categoryId }
+      });
+
+      if (!getCategory) {
+        return res.status(404).send({
+          success: false,
+          message: 'Category does not exits',
+        });
+      }
+      const article = await Article.create({
+        title: req.body.title,
+        body: req.body.body,
+        slug: articleSlug,
+        published: req.body.published,
+        userId: req.body.userId,
+        categoryId: req.body.categoryId
+      });
+
+      return res.status(201).send({
+        success: true,
+        message: 'Article created successfully',
+        article
+      });
+    } catch (err) {
+      return next(err);
+    }
+  }
+
+  /**
+   * @param {object} req - Request sent to the router
+   * @param {object} res - Response sent from the controller
+   *  @param {object} next - Error handler
+   * @param {object} id
+   * @returns {object} - object representing response message
+   */
+  static async getOneArticle(req, res, next) {
+    const { id } = req.params;
+    try {
+      const getArticle = await Article.findOne({
+        where: { id }
+      });
+
+      if (!getArticle) {
+        return res.status(404).json({
+          success: 'false',
+          message: 'Article not found',
+        });
+      }
+      return res.status(200).json({
```

Comment:
You should follow the convention in the readme file. Getting a single article should get this
```

{
  "article": {
    "slug": "how-to-train-your-dragon",
    "title": "How to train your dragon",
    "description": "Ever wonder how?",
    "body": "It takes a Jacobian",
    "tagList": ["dragons", "training"],
    "createdAt": "2016-02-18T03:22:56.637Z",
    "updatedAt": "2016-02-18T03:48:35.824Z",
    "favorited": false,
    "favoritesCount": 0,
    "author": {
      "username": "jake",
      "bio": "I work at statefarm",
      "image": "https://i.stack.imgur.com/xHWG8.jpg",
      "following": false
    }
  }
}
```